src/layer/tile/GridLayer.js

```diff
@@ -891,6 +908,27 @@ export const GridLayer = Layer.extend({
 		}
 	},
 
+	_tileUpdate(tile, err, el) {
```

Comment:
This method is used only once (in `refreshTiles()`) and therefore I'd be tempted to hoist it in there, converting it into an anonymous function.